lua/entities/terminator_nextbot_fakeply.lua

```diff
@@ -99,4 +99,25 @@ function ENT:GetAimVector()
     end
 
     return dir
-end
\ No newline at end of file
+end
+
+
+-- randomize bodygroups and skins
+function ENT:Initialize()
+    BaseClass.Initialize( self )
+    self:Appearance()
+end
+
+function ENT:Appearance()
+    local model = self:GetModel()
+    if not model then return end
+    
+    self:SetSkin( math.random( 0, self:SkinCount() - 1 ) )
+    
+    for i = 0, self:GetNumBodyGroups() - 1 do
+        local count = self:GetBodygroupCount( i )
+        if count > 1 then
+            self:SetBodygroup( i, math.random( 0, count - 1 ) )
+        end
+    end
+end
```

Comment:
Just style, and use a continue
```suggestion
function ENT:Appearance()
    local model = self:GetModel()
    if not model then return end
    
    self:SetSkin( math.random( 0, self:SkinCount() - 1 ) )
    
    for i = 0, self:GetNumBodyGroups() - 1 do
        local count = self:GetBodygroupCount( i )
        if count <= 1 then continue end
 
        self:SetBodygroup( i, math.random( 0, count - 1 ) )

    end
end
```